deployment/docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM amazonlinux:latest
+
+# Tools
+RUN yum install -y shadow-utils curl git openssl11-devel tar gzip gcc make bzip2-devel bzip2-libs ncurses-devel ncurses-lib libffi libffi-devel readline-devel sqlite-devel pyliblzma xz xz-devel xz-libs
```

Comment:
Could we use a lightweight image that already contemplates those packages? E.g. python:3.8.6-slim or https://github.com/GoogleCloudPlatform/python-docs-samples/blob/HEAD/run/helloworld/Dockerfile